WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -195,11 +237,36 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
             }
 
             @Override
-            public void afterTextChanged(Editable s) {}
+            public void afterTextChanged(Editable s) {
+            }
         });
         updateRemainingCharsView(remainingCharsTextView, mCustomMessage, MAX_CHARS);
     }
 
+    private boolean isUsernameContainsDelimiter(String username) {
```

Comment:
Nit: may I suggest renaming this to something like `endsWithDelimiter(String str)`?
Currently, the "...Contains.." part gives the wrong impression on the functionality. Also, it's a plain string function so no need for using "username" anyway.